src/stories/Library/Lists/list-reservation/list-reservation.scss

```diff
@@ -18,6 +18,14 @@ $list-reservation-space-desktop: 24px;
   padding: $list-reservation-space-mobile;
   cursor: pointer;
 
+  h3 {
+    color: $color__text-primary-black;
```

Comment:
Very nice that you're using the variables 👏 💯

Reply:
Thank you @Adamik10 🙏🏻